src/config-map.ts

```diff
@@ -227,9 +225,25 @@ export class ConfigMap extends base.Resource implements IConfigMap {
     }
   }
 
+  /**
+   * Adds an annotation to the supplied object that contains the checksum of the
+   * config map contents.
+   * @param obj Resource object to add the annotation to
+   */
+  public addChecksumTo(obj: base.Resource) {
+    obj.metadata.addAnnotation(
+      `checksum/${this.node.id}`,
```

Comment:
`node.id` is unique only in a given scope, `node.addr` will give an app unique id.
 
```suggestion
      `cdk8s.io/config-map./${this.node.addr}.checksum`,
```